packages/synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperPlotNav.stories.tsx

```diff
@@ -56,11 +56,11 @@ const handlePlotClick = (event: QueryWrapperSynapsePlotRowClickEvent) => {
 export const Cards: Story = {
   args: {
     name: 'Tools',
-    sql: 'SELECT * FROM syn26438037',
+    sql: 'SELECT * FROM syn51730943',
```

Comment:
This was failing, so updated to the new view (which also has a few examples of RRIDs that should be autolinked)